tests/test_variables.py

```diff
@@ -74,6 +76,38 @@ def test_pex_vars_set():
   assert v._get_int('HELLO') is None
 
 
+def test_pex_get_kv():
+  v = Variables(environ={})
+  assert v._get_kv('HELLO') is None
+  assert v._get_kv('=42') is None
+  assert v._get_kv('TOO=MANY=COOKS') is None
+  assert v._get_kv('THIS=WORKS') == ['THIS', 'WORKS']
+
+
+def test_pex_from_rc():
+  with tempfile.NamedTemporaryFile(mode='w') as pexrc:
+    pexrc.write('HELLO=42')
+    pexrc.flush()
+    v = Variables(rc=pexrc.name)
+    assert v._get_int('HELLO') == 42
+
+
+def test_pexrc_precedence():
+  with tempfile.NamedTemporaryFile(mode='w') as pexrc:
+    pexrc.write('HELLO=FORTYTWO')
+    pexrc.flush()
+    v = Variables(environ={'HELLO': 42}, rc=pexrc.name)
+    assert v._get_int('HELLO') == 42
+
+
+def test_rc_ignore():
+  with tempfile.NamedTemporaryFile(mode='w') as pexrc:
+    pexrc.write('HELLO=FORTYTWO')
+    pexrc.flush()
+    v = Variables(environ={'PEX_IGNORE_RC_FILES': True, 'HELLO': 42}, rc=pexrc.name)
+    assert v._get_int('HELLO') == 42
```

Comment:
This does not show ignore worked since 42 also wins by precedence if not ignored.

Fixing this test and optionally adding 2 more tests, one for .pexrc and one for ~/.pexrc .pexrc precedence would be great if you're around and have time to do this today.  If not I'll merge this in at 1pm Mountain and follow up with these before release later in the afternoon.

All the rest lgtm.